daemon/indexing/listener/listener.js

```diff
@@ -136,6 +139,26 @@ async function liveTracking(config) {
   check()
 }
 
+function getLastBlock(config){
+  if(config.continueFile == undefined || !fs.existsSync(config.continueFile)){
+    return 0
+  }
+  const json = fs.readFileSync(config.continueFile, {encoding:"utf8"})
+  const data = JSON.parse(json)
+  if(data.lastLogBlock){
```

Comment:
style nit: space before parenthesis ? it seems that is the style we use in origin-js repo ?